netstandard/pkg/NETStandard.Library.pkgproj

```diff
@@ -50,7 +50,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.targets))\Directory.Build.targets" />
 
   <PropertyGroup>
-    <OutDir>$(BaseOutputPath)shims/$(ShimRelOutputPath)</OutDir>
+    <OutDir>$(ArtifactsBinDir)shims/$(ShimRelOutputPath)</OutDir>
```

Comment:
Is this the right place to put the package?

Reply:
The package still gets placed in `artifacts\packages`, this defines the OutDir for the projects that this proj has as `ProjectReferences`.